tests/contracts/rip7560/TestPostOpPaymaster.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.25;
+
+import "../Stakable.sol";
+import "./TestAccount.sol";
+import {IRip7560Paymaster} from "./interface/IRip7560Paymaster.sol";
+import "../ValidationRules.sol";
+
+contract TestPostOpPaymaster is IRip7560Paymaster {
+    using ValidationRules for string;
+    uint256 public counter;
+    constructor() payable {}
+
+    function validatePaymasterTransaction(
+        uint256 version,
+        bytes32 txHash,
+        bytes calldata transaction)
+    external
+    returns (
+        bytes memory validationData
+    ){
+        RIP7560TransactionStruct memory txStruct = RIP7560Utils.decodeTransaction(version, transaction);
+        return RIP7560Utils.paymasterAcceptTransaction(txStruct.signature, 1, type(uint48).max - 1);
```

Comment:
"unlimited time" should be "0,0". no need to do type(uint48).max
(not only in this test, but in all tests)

Reply:
I did `1` and `type(uint48).max` in tests so that I can easily spot the values as a hex string. 
"All zeroes" is considered as "no limits" in the code:
```
if validUntil == 0 && validAfter == 0 {
	return nil
}
```